fastchat/eval/eval_gpt_review.py

```diff
@@ -42,15 +47,16 @@ def parse_score(review):
         if len(sp) == 2:
             return [float(sp[0]), float(sp[1])]
         else:
-            print('error', review)
-            return [-1, -1]
+            raise Exception('Invalid score pair.')
     except Exception as e:
-        print(e)
-        print('error', review)
+        logging.error(e)
```

Comment:
try to combine all error in a single string, so the error message will be (more likely) atomic

Reply:
fixed!